back-end/src/sciler/handler/handler_test.go

```diff
@@ -1204,11 +1204,18 @@ func TestLimitRule(t *testing.T) {
 				"status": true},
 		},
 	})
-	communicatorMock.On("Publish", "front-end", string(messageEventStatus), 3)
+
 	communicatorMock.On("Publish", "front-end", string(messageStatus), 3)
+	communicatorMock.On("Publish", "front-end", string(messageEventStatus), 3)
 	communicatorMock.On("Publish", "controlBoard", mock.Anything, 3)
+	for _, expectedCall := range communicatorMock.ExpectedCalls {
+		fmt.Println("exp:" + fmt.Sprint(expectedCall))
+	}
 	handler.msgMapper(msg)
-	communicatorMock.AssertNumberOfCalls(t, "Publish", 2)
+	for _, call := range communicatorMock.Calls {
```

Comment:
there is no longer an assert, and printing in tests isn't very useful in my opinion